src/main/java/seedu/address/logic/parser/ClientNoteEditCommandParser.java

```diff
@@ -54,7 +54,8 @@ public ClientNoteEditCommand parse(String restOfCommand) throws ParseException {
             newNote.setTags(tags);
             return new ClientNoteEditCommand(targetClientIndex, targetClientNoteIndex, newNote);
         } catch (ParseException pe) {
-            throw new ParseException(MESSAGE_INVALID_COMMAND_FORMAT);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    ClientNoteEditCommand.MESSAGE_USAGE), pe);
```

Comment:
what does the pe do?

Reply:
I reverted to this because as a fix to #272. `pe` is useless here, removed it.